build/python/backend/Dockerfile

```diff
@@ -110,6 +113,11 @@ RUN mkdir jtr && cd jtr && git init && git remote add origin https://github.com/
   && cp -Tr /jtr/run/ /jtr && rm -rf /jtr/run \
   && chmod -R 777 /jtr
 
+
+# Install Tesseract from source
+RUN git clone https://github.com/tesseract-ocr/tesseract.git && cd tesseract && ./autogen.sh && ./configure && make && make install && ldconfig && make training && make training-install
```

Comment:
We were just installing whatever `tesseract-ocr` `apt-get` would give us before. This still just installs the latest, but will be much more updated. We can tie this down to a version if we want to.

Reply:
We should clone a specific version -- I know that's different than what we had before, but we don't want to be on the bleeding edge without realizing it (ideally all versions are fixed, but at least `apt-get` is was getting released versions).

This is very similar to another case from the same file:
```mkdir jtr && cd jtr && git init && git remote add origin https://github.com/openwall/john.git && git fetch --depth 1 origin b5c10480f56ff1b5d76c6cbdaf9c817582ee2228 && git reset --hard FETCH_HEAD```

That's optimized to avoid retrieving the full commit history.

Reply:
sounds good 👍